From 9f94f774c1ef773fac8fd3415dc47a49efb0daaf Mon Sep 17 00:00:00 2001
From: Xiang Xu <xuxiang@eswincomputing.com>
Date: Wed, 13 Aug 2025 14:11:23 +0800
Subject: [PATCH 465/541] feat:update pin header function of ce1

Changelogs:
1. pinctrl  delete GPIO73 and GPIO87
2. SPI1 CS1(GPIO41) func sel to gpio

Signed-off-by: Xiang Xu <xuxiang@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/eic7700-ce1.dtsi | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-ce1.dtsi b/arch/riscv/boot/dts/eswin/eic7700-ce1.dtsi
index 91c9ca2f7145..d025773aa21d 100644
--- a/arch/riscv/boot/dts/eswin/eic7700-ce1.dtsi
+++ b/arch/riscv/boot/dts/eswin/eic7700-ce1.dtsi
@@ -822,10 +822,9 @@ &pinctrl {
 	status = "okay";
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_gpio18_default &pinctrl_gpio19_default &pinctrl_gpio20_default &pinctrl_gpio21_default
-			&pinctrl_gpio39_default &pinctrl_gpio40_default &pinctrl_gpio70_default &pinctrl_gpio73_default
+			&pinctrl_gpio39_default &pinctrl_gpio40_default &pinctrl_gpio41_default &pinctrl_gpio70_default
 			&pinctrl_gpio74_default &pinctrl_gpio75_default &pinctrl_gpio76_default &pinctrl_gpio79_default
-			&pinctrl_gpio80_default &pinctrl_gpio81_default &pinctrl_gpio82_default &pinctrl_gpio85_default
-			&pinctrl_gpio87_default >;
+			&pinctrl_gpio80_default &pinctrl_gpio81_default &pinctrl_gpio82_default &pinctrl_gpio85_default>;
 
     /* pin header default function for GPIO
        SPI1 (CS0,SCLK,MOSI,MISO,D2,D3,CS1): GPIO 35,36,37,38,39,40,41
@@ -854,7 +853,7 @@ pin : function
       22 : GPIO40
       23 : SPI1 CLK
       24 : SPI1 CS0
-      26 : SPI1 CS1
+      26 : SPI1 CS1(GPIO41)
       27 : I2C4 SDA
       28 : I2C4 SCL
       29 : GPIO80
-- 
2.54.0

